e2e/ansible/playbooks/hyperconverged/test-resize-jiva-volume/resize-cleanup.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Delete percona deployment
+  shell: kubectl delete -f "{{ percona_def }}"
+  args:
+    executable: /bin/bash
+  delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+
+- name: Delete the percona def copied
+  shell: rm -rf "{{ percona_def }}"
+  args:
+    executable: /bin/bash
+  delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
```

Comment:
Are we checking if the PV is getting deleted as part of the cleanup check ? Also, there doesn't seem to a confirmation check to see app has been removed as well !